src/main/java/de/hpi/bpt/argos/storage/PersistenceAdapterImpl.java

```diff
@@ -249,6 +249,41 @@ public Entity getMappingEntity(String sqlQuery) {
 	 * {@inheritDoc}
 	 */
 	@Override
+	public List<Event> getEvents(long entityOwnerId) {
+		Session session = databaseAccess.getSessionFactory().openSession();
+		Transaction transaction = session.beginTransaction();
+
+		Query<Event> query = session.createQuery("FROM EventImpl event "
+						+ "WHERE event.entityId = :entityOwnerId",
+				Event.class)
+				.setParameter("entityOwnerId", entityOwnerId);
+
+		return databaseAccess.getArtifacts(session, query, transaction, query::list, new ArrayList<>());
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public boolean getExistsEvent(String sqlQuery) {
+		if (sqlQuery == null || sqlQuery.length() == 0) {
+			return false;
+		}
+
+		Session session = databaseAccess.getSessionFactory().openSession();
+		Transaction transaction = session.beginTransaction();
+
+		Query query = session.createNativeQuery(sqlQuery);
+
+		String result = databaseAccess.getArtifacts(session, query, transaction, query::getSingleResult, "").toString();
+
+		return result.equals("True");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the "True" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1132)